include/deal.II/lac/affine_constraints.templates.h

```diff
@@ -57,6 +57,8 @@
 #include <boost/serialization/complex.hpp>
 #include <boost/serialization/utility.hpp>
 
+#include <Kokkos_View.hpp>
```

Comment:
That's a private include, see https://github.com/kokkos/kokkos/blob/f7c6ab6afcfe7d3d25ec9e9a055ad0f774e3743e/core/src/Kokkos_View.hpp#L17-L21.

Reply:
It is a known problem unfortunately. This files https://github.com/kokkos/kokkos/pull/7320/files tries to export that information in format that clang-format can understand but they are still some issues. Anyway, you can see in the file that for `Kokkos::View`, you need `Kokkos_Core.hpp`

Reply:
Ah, hm. Another reason to just throw everything into a single Kokkos module :-)

I assume that for `Kokkos_ExecPolicy.hpp` and `Kokkos_Parallel.hpp` I should then also just use `Kokkos_Core.hpp`?

Reply:
Yes, these files are not useful on their own.